cbor/src/test/java/tools/jackson/dataformat/cbor/mapper/BiggerDataTest.java

```diff
@@ -80,9 +81,13 @@ static class Area {
     /**********************************************************
      */
 
+	private final ObjectMapper MAPPER = JsonMapper.builder()
+			.disable(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES)
+			.build();
+
     public void testReading() throws Exception
 	{
-		Citm citm0 = jsonMapper().readValue(getClass().getResourceAsStream("/data/citm_catalog.json"),
+		Citm citm0 = MAPPER.readValue(getClass().getResourceAsStream("/data/citm_catalog.json"),
```

Comment:
Note that for `jackson-databind` I fixed this by changing POJO to have `Integer` for specific POJO field (from `int`) for case where needed.

But I'll merge this for now just to get builds going.